high-score-tables/src/components/App/App.jsx

```diff
@@ -1,16 +1,59 @@
-import HighScoreTable from "@/components/HighScoreTable/HighScoreTable.jsx";
-import scores from "@/data/scores.json";
 import "./App.scss";
+import "../scores.js";
+import allCountryScores from "../scores.js";
 
-const App = () => (
-  <div className="app">
-    <header className="app__header">
-      <h1 className="app__heading">High Scores</h1>
-    </header>
-    {scores.map(({ name, scores }, index) => (
-      <HighScoreTable country={name} scores={scores} key={index} />
-    ))}
-  </div>
-);
+function byName(a, b) {
+  if (a.name < b.name) return -1;
+  if (a.name > b.name) return 1;
+  return 0;
+}
+
+function App() {
+  allCountryScores.sort(byName);
+  return (
+    <>
+      <h1>HIGH SCORES PER COUNTRY</h1>
+      {allCountryScores.map((country, index) => (
+        <table
+          style={{
+            border: "1px solid black",
+            borderSpacing: "10px",
+            width: "100%",
+          }}
+        >
+          <thead>
+            <tr>
+              <th scope="col" style={{ border: "2px solid red" }} key={index}>
+                {country.name}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <ul style={{ borderBottom: "2px solid blue" }} scope="col">
+                <td style={{ padding: "1px" }}>player</td>
+                <td style={{ paddingLeft: "40px" }}>score</td>
+              </ul>
+            </tr>
+          </tbody>
+          <tbody>
+            <tr>
+              {country.scores.map((score, index) => (
+                <ul
+                  style={{ borderBottom: "1px solid red" }}
+                  scope="col"
+                  key={index}
+                >
+                  <td style={{ marginRight: "1px" }}>{score.n}</td>
+                  <td style={{ marginLeft: "90px" }}>{score.s}</td>
+                </ul>
+              ))}
+            </tr>
+          </tbody>
+        </table>
+      ))}
+    </>
+  );
+}
```

Comment:
Great Job!

If you wanted another challenge I would encourage you to look into importing individual components, this will be crucial when you start working with larger files.

Here is an article from the React team on this area https://react.dev/learn/importing-and-exporting-components

This can help in a few areas:

1. Allow you to create reusable components without duplicating code.
2. Cleanup larger files and make your code easier to read.

To use your work as an example, your App function could be refactored to look like: 

```
function App() {
  allCountryScores.sort(byName);

  return (
    <>
      <h1> HIGH SCORES PER COUNTRY </h1>

      {allCountryScores.map( ( country, index ) => (
        <ScoreTable country={country} index={index} />
      ) )}
    </>
  );
}
```